### PR TITLE
Address some TODOs

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Address some TODOs, in rand_basic_output() NTs are now added as feature already

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

### CODE REVIEW

File: sdk/src/types/block/rand/output/mod.rs [hunk 1]
```diff
@@ -43,7 +43,6 @@ pub fn rand_output_id() -> OutputId {
 
 /// Generates a random [`BasicOutput`](BasicOutput).
 pub fn rand_basic_output(token_supply: u64) -> BasicOutput {
-    // TODO: Add `NativeTokens`
```
Comment (thibault-martinez) [inline sdk/src/types/block/rand/output/mod.rs:46]: Because it's supposedly done in rand_allowed_features ?

Comment (Thoralf-M) [reply sdk/src/types/block/rand/output/mod.rs:46]: Yes, already done like this, read PR description :trollface:

Comment (thibault-martinez) [reply sdk/src/types/block/rand/output/mod.rs:46]: oopsie

**Review by thibault-martinez — APPROVED** [review]

File: sdk/src/client/secret/ledger_nano.rs [hunk 2]
```diff
@@ -145,7 +145,6 @@ impl SecretManage for LedgerSecretManager {
     ) -> Result<Vec<ed25519::PublicKey>, Self::Error> {
         // need an update on the ledger C lib
         todo!();
-        //
```
Comment (kwek20) [inline sdk/src/client/secret/ledger_nano.rs:148]: thats not addressing the todo :D

**Review by kwek20 — APPROVED** [review]

**Pull request merged**